paymentsheet-example/src/androidTestDebug/java/com/stripe/android/test/core/PlaygroundTestDriver.kt

```diff
@@ -296,7 +298,27 @@ class PlaygroundTestDriver(
 
                 blockUntilAuthorizationPageLoaded()
 
-                authorizeAction.click()
+                if(authorizeAction.exists()){
+                    authorizeAction.click()
+                }
+                // Failure isn't showing the same way each time.
```

Comment:
Could we maybe consolidate these branches? They look mostly the same minus the labels. Won't block the approval on it though.